build.sh

```diff
@@ -37,19 +75,34 @@ elif [ "$1" == "release" ]; then
     git submodule update --init --checkout
     git remote add pcengines https://github.com/pcengines/coreboot.git
     git fetch pcengines
-    git checkout $1
+    git checkout -f $1
     git submodule update --init --checkout
+
+    check_if_legacy $(git describe --tags --abbrev=0 ${1})
+    legacy=$?
+
     cd ../..
 
     VERSION=$1
     OUT_FILE_NAME="$2_${VERSION}.rom"
 
     # remove tag|branch from options
     shift
-    docker run --rm -it -v $PWD/release/coreboot:/home/coreboot/coreboot  \
-        -v $PWD/scripts:/home/coreboot/scripts pcengines/pce-fw-builder:latest \
-        /home/coreboot/scripts/pce-fw-builder.sh $*
 
+    if [ "$legacy" == 1 ]; then
```

Comment:
What about legacy branches? They are not tagged so they will not be distinguished by `check_if_legacy` function